yargs.js

```diff
@@ -241,7 +249,7 @@ function Yargs (processArgs, cwd, parentRequire) {
 
     if (typeof keys === 'number') {
       if (!options.demanded._) options.demanded._ = { count: 0, msg: null, max: max }
-      options.demanded._.count = keys
+      options.demanded._.count = keys // + context.commands.length
```

Comment:
this comment is accurate? assume it's here to explain the value of `options.demanded._.count`?